### ISSUE TITLE
Uploading non published files

### ISSUE THREAD

**Issue by slrslr** [zlatinb/muwire#58]
Muwire 0.8.7 is uploading files from folder for which i have not clicked Publish button.
I am not sure what is the use of the Publish button.
Maybe without it, the files i selected to share will not be found in search by others.
I am guessing that people are downloading Non published files, because they have browsed my file list and they are allowed to see such files and DL these anyway. If that is the case i am unsure how needed is the Publish button, why not remove it entirely, it can just cause people accidentally not share their data because they were thinking adding them to "Share" / Library is enough.

**Comment by zlatinb** [comment]
Every MuWire user has their own "File Feed".  When you click "Publish", you publish files to your file feed and anyone who subscribes to your feed gets to see them.

Users can download files which are not published.  It is enough to share (and hash) a file for others to be able to download it from you.

More information on File Feeds and a video available here: https://muwire.com/file-feeds.html

**Comment by slrslr** [comment]
"Publish" button can have many meaning, if it is "Publish to my feed", then one is wondering what is that feed and what is the purpose. The Feed tab though only shows third party feeds, so i see no overview of my feed or any description on how that feed works. But maybe it become apparent once i subscribe to someone's feed and it is visible on Feed tab. I am a newbie so i have felt no need to subscribe someone's feed yet.

**Comment by zlatinb** [comment]
I agree, there are many things that can be improved.  Maybe by default everyone should be subscribed to their own feed so they can easily see which files they've published.  I'll try to do something like that for the next release.
